rpm/libtasn1.spec

```diff
@@ -56,19 +60,34 @@ Man and info pages for %{name}.
 %setup -q -n %{name}-%{version}/%{name}
```

Comment:
You probably should change this to use %autosetup. Now the patches are not applied because there is no %patch macros.